### PR TITLE
Upgrade cluster test

### PR DESCRIPTION
**Pull request by maathor**

### CODE REVIEW

File: tests/aws/aws_kubernetes.rs [hunk 1]
```diff
@@ -20,39 +20,70 @@ use std::io::{BufReader, Read};
...
 use std::process::Command;
 use std::{env, fs};
-use test_utilities::aws::AWS_KUBERNETES_VERSION;
+use test_utilities::aws::{
+    aws_access_key_id, aws_default_region, aws_secret_access_key, AWS_KUBERNETES_VERSION,
+};
 
-pub const QOVERY_ENGINE_REPOSITORY_URL: &str = "CHANGE-ME";
+pub const QOVERY_ENGINE_REPOSITORY_URL: &str = "https://github.com/Qovery/engine.git";
 pub const TMP_DESTINATION_GIT: &str = "/tmp/qovery-engine-master/";
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:28]: please use main instead

File: tests/aws/aws_kubernetes.rs [hunk 2]
```diff
@@ -20,39 +20,70 @@ use std::io::{BufReader, Read};
...
-            password: GIT_TOKEN.to_string(),
-        }),
+        // this repo is public !
+        credentials: None,
         commit_id: "".to_string(),
         dockerfile_path: "".to_string(),
     };
     fs::create_dir_all(tmp_dir.clone());
-    info!("Cloning qovery-engine repository");
+    info!("Cloning old engine repository");
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:47]: Why don't you just copy/paste the current one and checkout on the main branch instead?

File: tests/aws/aws_kubernetes.rs [hunk 3]
```diff
@@ -69,32 +100,53 @@ fn create_and_upgrade_cluster_from_master_branch() {
...
     };
     assert!(env::set_current_dir(&current_path).is_ok());
     create_eks_cluster_in_eu_west_3();
     delete_eks_cluster_in_eu_west_3();
 }
 
 #[test]
+#[ignore]
 fn create_eks_cluster_in_us_east_2() {
+    // PERMIT_CLUSTER_CREATION env variable prevent you to not spend money on unnecessary cluster creation
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:145]: Isn't it redudant with recent dry_run support?

File: tests/aws/aws_kubernetes.rs (hunk 2 above)
Comment (maathor) [reply tests/aws/aws_kubernetes.rs:47]: will put a enhancement on it

**Review by deimosfr — CHANGES_REQUESTED** [review]
You PR contains commits that should never be there. Please prefer fast forward commit instead

**Pull request closed without merging**